libs/aio/odc/aio/__init__.py

```diff
@@ -292,6 +292,82 @@ def default_guide(url, depth, base):
     return step
 
 
+def s3_uri_to_stream(uri: str, skip_check: bool):
```

Comment:
I think `s3_uri_to_stream` is a confusing name, I would assume that it streams content of some large file as bytes/byte-chunks or something like that, from name alone

Reply:
`s3_uri_to_generator` is a better name ? Naming stuff is hard.

Reply:
it's not `uri`, it's `glob`, and what is `stream`?

Reply:
`s3_glob_to_keys` is a better name ?